packages/shared/src/components/auth/AuthenticationBanner.tsx

```diff
@@ -7,13 +7,17 @@ import { AuthTriggers } from '../../lib/auth';
 import { MemberAlready } from '../onboarding/MemberAlready';
 import { useAuthContext } from '../../contexts/AuthContext';
 import { BottomBannerContainer } from '../banners';
+import { useFeature } from '../GrowthBookProvider';
+import { feature } from '../../lib/featureManagement';
+import { PostPageOnboarding } from '../../lib/featureValues';
 
 const Section = classed('div', 'flex flex-col w-[23.25rem]');
 export const authGradientBg =
   'bg-theme-bg-primary bg-gradient-to-l from-theme-overlay-active-cabbage from-0% to-theme-overlay-active-onion to-100%';
 
 export function AuthenticationBanner(): ReactElement {
   const { showLogin } = useAuthContext();
+  const postPageOnboarding = useFeature(feature.postPageOnboarding);
```

Comment:
Why do you prop drill it and not directly access the feature on the last component?
Seems we don't really do anything else with it right?

Reply:
We do with AuthOptions. The onboarding auth options show in the banner and in the onboarding page. So if I use it in auth options we would be enrolling users landing on the onboarding page, not only post page.

Reply:
Oh got it, ok :)